galasa-extensions-parent/dev.galasa.ecosystem.docker.builder/src/main/java/dev/galasa/ecosystem/docker/builder/DockerEcosystemBuilder.java

```diff
@@ -0,0 +1,525 @@
+package dev.galasa.ecosystem.docker.builder;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.util.Map;
+import java.util.Scanner;
+
+import org.yaml.snakeyaml.Yaml;
+
+public class DockerEcosystemBuilder {
+
+	public static void main(String[] args) throws FileNotFoundException {
+		
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Welcome to the Galasa Ecosystem manager for Docker \n");
+		System.out.println("If you have not run this program before, default values for the configuration have been supplied \n");
+		System.out.println("Here is your current configuration for the Ecosystem: \n");
+		String configLocation = "classes/";
+		
+		try {
+			BufferedReader dis = new BufferedReader(new FileReader(configLocation + "config.yml"));
+			String line;
+			while((line = dis.readLine()) != null)
```

Comment:
The printing of the options should be formatted a nicer way.   Also each option should be labelled with a number.   Ie.  1. Hostname of the docker server = xxxx.xxx.xxxx.
So when the user wants to change the parameter,  they select 1 instead of hostname.